pkg/list/list_workflows.go

```diff
@@ -0,0 +1,211 @@
+package list
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"strings"
+
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/cloudposse/atmos/internal/exec"
+	"github.com/cloudposse/atmos/pkg/ui/theme"
+	"github.com/samber/lo"
+
+	"github.com/cloudposse/atmos/pkg/schema"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	FormatTable = "table"
+	FormatJSON  = "json"
+	FormatCSV   = "csv"
+)
+
+// ValidateFormat checks if the given format is supported
+func ValidateFormat(format string) error {
+	if format == "" {
+		return nil
+	}
+	validFormats := []string{FormatTable, FormatJSON, FormatCSV}
+	for _, f := range validFormats {
+		if format == f {
+			return nil
+		}
+	}
+	return fmt.Errorf("invalid format '%s'. Supported formats are: %s", format, strings.Join(validFormats, ", "))
+}
+
+// LineEnding returns the appropriate line ending for the current OS
+func LineEnding() string {
+	if runtime.GOOS == "windows" {
+		return "\r\n"
+	}
+	return "\n"
+}
```

Comment:
Seems like something that ought to be handled natively in go. Either way should not be something unique to workflows.